fastapi_injector/request_scope.py

```diff
@@ -19,6 +26,21 @@
 _request_id_ctx: ContextVar[uuid.UUID] = ContextVar("request_id")
 
 
+class RequestScopeOptions:
+    """
+    Defines the behavioural options for a request scope.
+
+    Available options are:
+      * enable_cleanup
+            If True, dependencies that were created from the request scope will be
+            cleaned up when the scope is exited. Only dependencies that implement one
+            of the context manager protocols will be considered for cleanup.
+    """
+
+    def __init__(self, **kwargs) -> None:
+        self.enable_cleanup = kwargs.get("enable_cleanup", False)
```

Comment:
Please make this a `dataclass` or similar, so that it doesn't have to be passed as black-box kwargs.

Reply:
Done